### PR TITLE
operators couchbase-enterprise (2.3.0)

### PR DESCRIPTION
**Pull request by tahmmee**

### CODE REVIEW

File: operators/couchbase-enterprise/2.3.0/metadata/annotations.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  operators.operatorframework.io.bundle.channel.default.v1: stable
+  operators.operatorframework.io.bundle.channels.v1: stable
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: couchbase-enterprise
+
+  # Annotations for testing.
+  operators.operatorframework.io.test.mediatype.v1: scorecard+v1
+  operators.operatorframework.io.test.config.v1: tests/scorecard/
```
Comment (mvalarh) [inline operators/couchbase-enterprise/2.3.0/metadata/annotations.yaml:12]: @tahmmee Since there are no tests in this bundle. Please remove these 2 lines

**Review by mvalarh — CHANGES_REQUESTED** [review]

**Comment by mvalarh** [comment]
@tahmmee please make image available for public
```
$ podman pull registry.connect.redhat.com/couchbase/operator:2.3.0-1
Trying to pull registry.connect.redhat.com/couchbase/operator:2.3.0-1...
Error: initializing source docker://registry.connect.redhat.com/couchbase/operator:2.3.0-1: unable to retrieve auth token: invalid username/password: unauthorized: Please login to the Red Hat Registry using your Customer Portal credentials. Further instructions can be found here: https://access.redhat.com/RegistryAuthentication
```

**Comment by tahmmee** [comment]
Hi @mvalarh any insight to why upgrade is failing here?  going from 2.1.0 -> 2.3.0 does require replacement of CRD's, but not sure if that's happening.

**Comment by J0zi** [comment]
retesting

**Pull request merged**